azurerm/internal/services/databricks/databricks_workspace_resource.go

```diff
@@ -184,6 +184,8 @@ func resourceDatabricksWorkspaceCreateUpdate(d *schema.ResourceData, meta interf
 	customParamsRaw := d.Get("custom_parameters").([]interface{})
 	customParams := expandWorkspaceCustomParameters(customParamsRaw)
 
+	// Including the Tags in the workspace parameters will update the tags on
+	// the workspace only
```

Comment:
this seems like an API bug? could we open a issue on the sdk/specs and link to it here?

Reply:
it kind of is, although tbh this is another case where it'd be beneficial to split the Create/Update here longer-term?

Reply:
This totally is an API bug and I have opened issue [14571](https://github.com/Azure/azure-sdk-for-go/issues/14571) to track this bug.